report-03/threat model.wiki

```diff
@@ -40,7 +40,7 @@ This document outlines the OBPP threat model used to develop [[criteria.md|crite
 |Avoid constructing transactions that contain inputs from more than one identity/account
 |III A 1 a <br/> III A 2 a <br/> III A 3 a
 |-
-|rowspan="3"|A blockchain observer can derive the type of wallet used to create a transaction by observing idiosyncrasies in the manner in which the transaction was composed
+|rowspan="3"|A blockchain observer can derive the type of wallet used to create a transaction by observing idiosyncrasies in the manner in which the transaction was composed, such as inputs/output ordering, output or fee size, or script composition.
```

Comment:
reminder to self: one of the attacks we talked about previously was to record suggested fees by a variety of dynamic fee services, and then compare transaction fees on the blockchain to that record to attempt to fingerprint a client.